api/deploy_template.yaml

```diff
@@ -63,6 +63,8 @@ objects:
               secretKeyRef:
                 name: service-portal-api-secrets
                 key: encryption-key
+          - name: ENFORCE_TENANCY
+            value: "true"
```

Comment:
Do we ever want to make this not true? Should this be a parameter?

Reply:
Also the quotes are not necessary.

Reply:
@carbonin there is no reason it wouldn't be true.  If yes it would be rails console specific and the gem for tenancy has numerous tools to handle turning it off per session.

The quotes don't matter.  The code just looks for the environment variable.
If you would just rather the quotes weren't in the yaml, I can make that change.

Reply:
@carbonin - The thought was to add this variable so it isn't forgotten in other builds with the understanding it will be moved into a settings or config Gem at some point.  We want it enabled for any Openshift environment ( local environments wouldn't need it. ) The same env variable is part of Topology ( under discussion in [this](https://github.com/ManageIQ/topological_inventory-api/pull/104/files) PR ) 
Moving it into the Dockerfile may make sense too, but I don't think that would be permanent either.